### ISSUE TITLE
Download for windows runtime is down

### ISSUE THREAD

**Issue by marca278** [kanbanik/kanbanik#26]

**Comment by jelkosz** [comment]
indeed, all of the runtimes are gone. Google decided to delete them even they were supposed to remain there indefinitely...
Sorry about that, I will try to restore that during the weekend.

**Comment by jelkosz** [comment]
So I was trying to create a new one but got discouraged in a meanwhile.  There is no good way how to make the portable versions of java and mongodb. With the introduction of docker for windows I think I will rather start publishing kanbanik as a docker container instead of the runtime. 

In meanwhile, you will need to install the dependencies by hand - explained here: https://github.com/kanbanik/kanbanik/wiki/Installation-Guide 

I will keep this issue opened for a while - in case you would have any issues installing anything, let me know here so I can assist.

**Comment by jelkosz** [comment]
Made some changes so kanbanik is now working well inside of a docker container. Available here:
https://hub.docker.com/r/jelkosz/kanbanik/

Also enriched the documentation on how to run it together with mongodb as docker container:
https://github.com/kanbanik/kanbanik/wiki/Installation-Guide#install-from-docker

It is only for linux so far mostly because I have not tested it at all on windows. Will try to add also the description for windows soon.
